src/Sdk/PasswordlessClient.cs

```diff
@@ -41,6 +39,15 @@ public PasswordlessClient(HttpClient client)
         _client = client;
     }
 
+    public async Task AddAliasAsync(AddAliasRequest request, CancellationToken cancellationToken)

```

Comment:
Include the docs you wrote on the interface.

```suggestion
    /// <inheritdoc/>
    public async Task AddAliasAsync(AddAliasRequest request, CancellationToken cancellationToken)

```